azurerm/internal/services/network/public_ip_resource.go

```diff
@@ -103,6 +103,18 @@ func resourcePublicIp() *pluginsdk.Resource {
 				}, true),
 			},
 
+			"sku_tier": {
```

Comment:
can we add this new property [to the documentation too](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/public_ip.html.markdown#L49-L73)?